postprocess/remove_crds_on_additional_instance.jsonnet

```diff
@@ -0,0 +1,15 @@
+local com = import 'lib/commodore.libjsonnet';
+local inv = com.inventory();
+local instance = inv.parameters._instance;
+local manifests_dir = std.extVar('output_path');
+
+local stripCRDs(obj) =
+  if instance == 'paralus-test' then
```

Comment:
This seems very specific. Can we do a parameter `install_crds` that defaults to `true` instead? Then we configure this more flexible in case we want more / other test instances at some point.